src/View/View.php

```diff
@@ -182,14 +181,6 @@ class View implements EventDispatcherInterface
      */
     protected $hasRendered = false;
 
-    /**
-     * List of generated DOM UUIDs.
-     *
-     * @var array
-     * @deprecated 3.7.0 The property is unused and will be removed in 4.0.0.
-     */
-    public $uuids = [];
```

Comment:
The `uuid()` method on `View` uses this property. I'm not confident we *need* that method either though.

Reply:
If someone needs it they can easily wrap it using a helper method :)

Reply:
I would much more like to see the model prefixing coming back for "id" values as the collisions on forms usually otherwise are always present somehow.

Reply:
@markstory I am pretty sure I removed the method too. Dunno how it didn't get committed :) Will update.

Reply:
`View::uuid()` removed.